system/ee/language/english/fieldtypes_lang.php

```diff
@@ -75,19 +75,19 @@
 
     'rel_ft_display_entry_id_desc' => 'When enabled, entry IDs will be displayed together with entry title inside the field.',
 
-    'rel_ft_max' => 'Maximum selection',
+    'rel_ft_max' => 'Maximum relationship',
 
     'rel_ft_max_desc' => 'Sets the maximum number of entries this field can be used to relate.',
 
     'rel_ft_max_error' => 'You can select no more than %d entries.',
 
-    'rel_ft_min' => 'Minimum selection',
+    'rel_ft_min' => 'Minimum relationship',
```

Comment:
possibly change to "Minimum number of related entries"

Reply:
Gut reaction is that I like this suggestion.